bigtable/admin.go

```diff
@@ -275,6 +275,44 @@ func (ac *AdminClient) CreateColumnFamily(ctx context.Context, table, family str
 	return err
 }
 
+// UpdateTableConf contains all of the information necessary to update a table with column families.
+type UpdateTableConf struct {
+	TableID string
+	// Set to true to make the table protected against data loss
+	// i.e. deleting the table, the column families in the table and the instance containing the table would be prohibited
+	DeletionProtection *bool
```

Comment:
Per a previous comment, I don't think this needs to be a pointer to a bool--it can just be a bool.